app/src/main/java/com/example/cocktails/ScrollingActivity.kt

```diff
@@ -35,8 +35,8 @@ import kotlinx.android.synthetic.main.filter_dialog.view.*
 
 @Parcelize
 data class Cocktail(val name: String, val type: String, val steps: Array<String>,
-                    val ingredients: Array<String>, val image: String, val isCustom: Boolean = false,
-                    val glass: Uri?): Parcelable
+                    val ingredients: Array<String>, val clipart: String, val image: String,
+                    val isCustom: Boolean = false, val glass: String?): Parcelable
```

Comment:
@shahafhermann temporary change, feel free to re rewrite it, but notice i added new clipart and image parameters as well

Reply:
No problem, that's probably the best solution.